app/controllers/registrations_controller.rb

```diff
@@ -21,4 +22,7 @@ def after_inactive_sign_up_path_for(_resource)
   def update_resource(resource, params)
     resource.update_without_password(params)
   end
+  
+  def public_new
```

Comment:
Do we still need this?

Reply:
No we don't. Thanks for pointing this out! @sreycodes